styles.css

```diff
@@ -0,0 +1,116 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap");
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+}
+body {
+  background: rgb(241, 245, 248);
+}
+.sidebar-toggle-btn {
+  font-size: 32px;
+  color: rgb(73, 166, 233);
+  float: right;
+  background-color: transparent;
+  border: 0;
+  margin-top: 20px;
+  margin-right: 20px;
+  cursor: pointer;
+  animation: zoom 2s ease-in-out infinite;
+}
+@keyframes zoom {
+  0% {
+    transform: scale(1);
+  }
+  30% {
+    transform: scale(1.2);
+  }
+  50% {
+    transform: scale(1.5);
+  }
+  100% {
+    transform: scale(1);
+  }
+}
+.sidebar {
+  width: 30%;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  background: rgba(139, 94, 94, 0.16);
+  visibility: hidden;
+}
+.show-sidebar {
+  visibility: visible;
```

Comment:
Are we able to click through this when it is not visible? We should be.
Using display none would not allow transitions to work
Consider toggling width from 0 to current size on show sidebar.

Reply:
Definitely, I'll resolve it.
I didn't consider it, as our webpage body didn't have any other components on the left side. That is why, I didn't check it out.
Thanks for your suggestion.